src/test/java/Utils.java

```diff
@@ -0,0 +1,45 @@
+import java.net.URL;
+import org.json.JSONObject;
+import java.util.Arrays;
+
+
+/**
+ * @version 1.0
+ * @author B. Clint Hall
+ */
+public class Utils {
```

Comment:
Thanks for making this class. Super helpful; should cut down a lot on code duplication.